README.md

```diff
@@ -117,16 +117,16 @@ Does nothing if versions are in sync.
 
 Environment config:
 
-- ``SERVER``: server URL (default: ``http://localhost:8888/v1``)
-- ``MEGAPHONE_HOST``: Push service host (eg.: ``push.services.mozilla.com``)
-- ``MEGAPHONE_AUTH``: Push service bearer token
+- ``SERVER``: Remote Settings server URL (default: ``http://localhost:8888/v1``)
+- ``MEGAPHONE_URL``: Megaphone service URL (eg.: ``https://megaphone.services.mozilla.com/v1``)
+- ``MEGAPHONE_AUTH``: Megaphone bearer token
 - ``BROADCASTER_ID``: Push broadcaster ID (default.: ``remote-settings``)
 - ``CHANNEL_ID``: Push channel ID (default.: ``monitor_changes``)
 
 Example:
 
 ```
-$ SERVER=https://settings.prod.mozaws.net/v1 MEGAPHONE_HOST="push.services.mozilla.com" MEGAPHONE_AUTH="a-b-c" python aws_lambda.py sync_megaphone
+$ SERVER=https://settings.prod.mozaws.net/v1 MEGAPHONE_URL="push.services.mozilla.com" MEGAPHONE_AUTH="a-b-c" python aws_lambda.py sync_megaphone
```

Comment:
The example `MEGAPHONE_URL` should be a URL (probably just the default from above).